### ISSUE TITLE
BUG: Crash on driver Seat

### ISSUE THREAD

**Issue by RickyLaChow** [VolmitSoftware/VehiclesPlus#73]
**Describe the bug**  
When i join on my car if i'm Driver, i have been Disconnected from the server

**To Reproduce**  
1. Open Vehicle Garage
2. Go Driver Seat 
3. And you get kicked out from the server, it say is crashed.

**Expected behavior**  
No Errors in Console, but when i get kicked out say the server is crashed and i get back in the fallback server.

**Server and Plugin Information**
 - VehiclesPlus Version: 2.2.4
 - Pro or Lite? Pro
 - Server Platform and Version [eg: **PaperSpigot** 1.16.1 #135]: PaperSpigot 1.16.5
 - Operating System (if applicable):

**Comment by RickyLaChow** [comment]
The problem is an incompatibility with ItemsAdder, if ItemsAdder is on my Plugins folder with LoneDev API when i join in a car i get kicked out from the server for "crash"

**Comment by LoneDev6** [comment]
Provide screenshots of the issue

**Comment by RickyLaChow** [comment]
> Provide screenshots of the issue.
> "crash" is not a valid reason, never heard of "crashed" as kick reason

![2021-02-08_18 50 30](https://user-images.githubusercontent.com/25041502/107260609-94173e80-6a3e-11eb-8f6b-eceffc5f25f4.png)

**Comment by LoneDev6** [comment]
I'm sure there is a reason about that kick, surely a bad packet or something.
You should check if you can enable more detailed Bungeecord logging to see which packet made the client disconnect

**Comment by RickyLaChow** [comment]
> I'm sure there is a reason about that kick, surely a bad packet or something.
> You should check if you can enable more detailed Bungeecord logging to see which packet made the client disconnect

How?

**Comment by stijnb1234** [comment]
> > I'm sure there is a reason about that kick, surely a bad packet or something.
> > You should check if you can enable more detailed Bungeecord logging to see which packet made the client disconnect
> 
> How?

Normally you can see the log in your Bungeecord console.

**Comment by RickyLaChow** [comment]
i don't have anything 

RolePlay: 
[19:24:37 INFO]: Rickylacio issued server command: /vehicle garage
[19:24:39 INFO]: Rickylacio lost connection: Disconnected
[19:24:39 INFO]: Rickylacio left the game

Bungee:
19:24:18 [INFO] [Rickylacio] <-> DownstreamBridge <-> [lobby] has disconnected
19:24:39 [INFO] [Rickylacio] <-> DownstreamBridge <-> [roleplay] has disconnected
19:24:39 [INFO] [Rickylacio] <-> ServerConnector [lobby] has connected

**Comment by RickyLaChow** [comment]
Probably problem of ResourcePack

**Comment by relavis** [comment]
Not a problem of the resource pack at all - resource packs are client side. This is an issue with ItemsAdder that we cannot fix. Please make an issue report with them.

**Comment by LoneDev6** [comment]
2. Does the issue still occurs if:
- remove itemsadder and keep VehiclesPlus?

1. Does the issue still occurs if:
- remove VehiclesPlus and keep ItemsAdder?

**Comment by LoneDev6** [comment]
> 
> 
> Not a problem of the resource pack at all - resource packs are client side. This is an issue with ItemsAdder that we cannot fix. Please make an issue report with them.

If answers for my two questions are both YES then it's not an ItemsAdder-only issue, we may have to collaborate and find the cause of the incompatibility.

**Comment by relavis** [comment]
>     1. Does the issue still occurs if:
> 
> 
>     * remove itemsadder and keep VehiclesPlus?
> 
> 
>     1. Does the issue still occurs if:
> 
> 
>     * remove VehiclesPlus and keep ItemsAdder?

Oh man, didn't even realize you're the dev of ItemsAdder - my bad. Will reopen.

I believe the user clarified that it occurs only when ItemsAdder is installed, and removing ItemsAdder solves the crash.

**Comment by LoneDev6** [comment]
No problem.
Anyway what I meant is that ItemsAdder doesn't cause any crash if VehiclesPlus is not installed.
Same thing the other way around.

So I think we have to find which packet is sent only when both plugins are installed. I sincerly have no idea as I don't own the plugin

**Comment by RickyLaChow** [comment]
> > ```
> > 1. Does the issue still occurs if:
> > 
> > 
> > * remove itemsadder and keep VehiclesPlus?
> > 
> > 
> > 1. Does the issue still occurs if:
> > 
> > 
> > * remove VehiclesPlus and keep ItemsAdder?
> > ```
> 
> Oh man, didn't even realize you're the dev of ItemsAdder - my bad. Will reopen.
> 
> I believe the user clarified that it occurs only when ItemsAdder is installed, and removing ItemsAdder solves the crash.

Yes the problem is when i have installed ItemsAdder, i give all informations
I contact Lone, and you for found a solutions for this incompatibility

**Comment by LoneDev6** [comment]
Added @relavis on Discord

**Comment by stijnb1234** [comment]
> Added @relavis on Discord

👍 Please update in this issue if anything changes.

**Comment by RickyLaChow** [comment]
Any news?

**Comment by LoneDev6** [comment]
![immagine](https://user-images.githubusercontent.com/27242001/115266411-79e19700-a138-11eb-9e06-443e64b9a0e9.png)

No answer from them so.. no

**Comment by stijnb1234** [comment]
@LoneDev6 I'm the new developer of this plugin. Can you please contact me on Discord? I'm curious to see what's been communicated with relavis about this issue.

**Comment by LoneDev6** [comment]
Join my discord server and open a ticket stating you're the developer of vehicles plus
https://support.plugin.ga/

**Comment by stijnb1234** [comment]
Received the plugin from @LoneDev6. Hoping to test it myself very soon, and fix it.
